samples/subsys/usb/uac2_explicit_feedback/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "../app.overlay"
+
+&pinctrl {
```

Comment:
Someone might try to use an Arduino shield with this sample, for whatever reason. Should [arduino_spi](https://github.com/zephyrproject-rtos/zephyr/blob/e57ad265fb4fd9f2fcd1237b8ee7897245b62fcd/boards/arm/nrf5340dk_nrf5340/nrf5340_cpuapp_common.dtsi#L216) node be deleted? @gmarull

Reply:
The same would then have to apply to https://github.com/zephyrproject-rtos/zephyr/blob/5273af6ba80a5919fd56e69d05b820beadb3ec0e/samples/drivers/i2s/echo/boards/nrf5340dk_nrf5340_cpuapp.overlay that uses the same pins for I2S as this sample.